### DISCUSSION TITLE
Convert Xiaomi LYWSD03MMC From Bluetooth To Tuya?

### DISCUSSION THREAD

**Discussion by Wistiii** [devbis/z03mmc#53] [Ideas]
Hello,
First...great job, it works well and easy with the OTA.
After a long revue on internet...  this little sensor  seems to be reluctant to Tuya or Smartlife App. Any project for that?

Thanks
W

**Comment by devbis** [comment]
It's hard to say 'cause I don't have Tyua gateways. Maybe model replacement to tuya-like will work. 
https://github.com/devbis/z03mmc/blob/master/src/sensorEpCfg.c#L25-L28

I can't check it, but you may try
